hyp3_rtc_gamma/rtc_sentinel.py

```diff
@@ -38,6 +38,18 @@
 from hyp3_rtc_gamma.xml2meta import sentinel2meta
 
 
+def get_product_name(granule_name, resolution=30, gamma0=True, power=True, filtered=False):
+    platform = granule_name[0:3]
+    beam_mode = granule_name[4:6]
+    datetime = granule_name[17:32]
+    g = 'g' if gamma0 else 's'
+    p = 'p' if power else 'a'
+    f = 'f' if filtered else 'n'
```

Comment:
I think I prefer parentheses in conditional assignment `result = (true_value if condition else false_value)`

Reply:
> I think I prefer parentheses in conditional assignment `result = (true_value if condition else false_value)`
> 

PyCharm and pylint will give "redundant parentheses" warnings for that